.pre-commit-config.yaml

```diff
@@ -37,4 +37,13 @@ repos:
     name: pylint
     entry: pylint
     language: system
-    types: [python]
+    types: [python
+
+]
+
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: 'v0.940'
+  hooks:
+  - id: mypy
+    args: [--no-strict-optional, --ignore-missing-imports]
```

Comment:
I actually find checking for optionals to be the most important use case for mypy - helps to avoid Null pointer errors. Can we keep it?